src/ophys_etl/modules/suite2p_registration/__main__.py

```diff
@@ -28,8 +30,29 @@ def run(self):
                                               "local build")
         self.logger.info(f"OPHYS_ETL_COMMIT_SHA: {ophys_etl_commit_sha}")
 
-        # register with Suite2P
+        # Get suite2p args.
         suite2p_args = self.args['suite2p_args']
+
+        if (suite2p_args['force_refImg'] and len(suite2p_args['refImg']) == 0):
+            # Use our own version of compute_reference to create the initial
+            # reference image used by suite2p.
+            self.logger.info('Creating initial reference image from '
+                             f'{suite2p_args["nimg_init"]} frames...')
+            intial_frames = load_initial_frames(
+                file_path=suite2p_args['h5py'],
+                h5py_key=suite2p_args['h5py_key'],
+                n_frames=suite2p_args['nimg_init'],)
+            # Create the initial reference image and store it in the
+            # suite2p_args dictionary. 8 iterations is the current default in
+            # suite2p.
+            suite2p_args['refImg'] = compute_reference(
+                frames=intial_frames,
+                niter=8,
```

Comment:
Hmmm... I think you're right. I have another ticket #394 where I'm going to investigate a dynamic stopping criteria but that is still going to have a relative tolerance and a max_iterations. Do you think should I put this in the schema for suite2p_wrapper or suite2p_registration? I'm leaning towards the latter.

Reply:
Butting in: I think it goes in suite2p_registration.

My interpretation of suite2p_wrapper is that it is a module that gives user-facing modules access to Suite2P. Since Suite2P doesn't know about your reference image generator, that is something that belongs in the user facing layer (suite2p_registration)

Reply:
Any name suggestions? `max_reference_iterations`?

Reply:
I am good with that.